packages/simm-validation/src/is-strong-password.ts

```diff
@@ -0,0 +1,87 @@
+import { regexValidate } from "./shared/constants/regex";
+
+interface Options {
+  minLength: number;
+  minLowercase?: number;
+  minUppercase?: number;
+  minNumbers?: number;
+  minSymbols?: number;
+}
+
+/**
+ * Check the total count of uppercase, lowercase, number, and symbol chars.
+ *
+ * @param {any} input The string to analyze.
+ * @returns {boolean} Returns analyzed result.
+ */
+const analyzePassword = (input: string) => {
+  let uppercaseCount = 0;
+  let lowercaseCount = 0;
+  let numberCount = 0;
+  let symbolCount = 0;
+
+  input.split("").forEach((char) => {
```

Comment:
dùng spread oparators đi e 
```suggestion
  [...test].forEach((char) => {
```